heat/core/tests/test_dndarray.py

```diff
@@ -1112,13 +1136,13 @@ def test_setitem_getitem(self):
         a[1:4, 1] = 1
         self.assertTrue((a[1:4, 1] == 1).all())
         self.assertEqual(a[1:4, 1].gshape, (3,))
-        self.assertEqual(a[1:4, 1].split, 0)
+        self.assertEqual(a[1:4, 1].split, None)
```

Comment:
so this is a more major change. can you add something to the changelog which mentions that slicing with an int in the split dimension will result in a split=None?

perhaps we should have a file which contains general usage notes for things like this?

Reply:
Added under `Breaking Changes`, and yes, I'll write something up for this one.